Update segment.mdx (#165)
launchdarkly/LaunchDarkly-Docs: src/content/topics/home/data-export/segment.mdx

published: true
---

> <Callout intent="primary">
>   <CalloutTitle>Experimentation Integration for Segment</CalloutTitle>
>   <CalloutDescription>
> 
> __If you're looking for the Experimentation integration for Segment, the documentation is located [here.](https://docs.launchdarkly.com/home/experimentation/segment)__
>     
>   </CalloutDescription>
> </Callout>
> 
> ## Overview
> 
> This topic explains how to create and test a Segment destination for Data Export.

Reviewer: ```suggestion
## Overview

This topic explains how to create and test a Segment destination for Data Export.

For information about the Experimentation integration for Segment, read [Segment](/home/experimentation/segment).
```

Author: Ah, I guess I'm not entirely familiar with the qualifications for a callout. I thought it might be more eye-catching as this page is the first one that pops up when searching for "Segment" in the search box whereas the linked page (Sending SEgment events to experiments) is quite a few below. I think the non-callout reference also works.

Reviewer: Good to know about the search. Hopefully that can be fixed once we have tagging in place. You've convinced me!

> This topic explains how to create and test a Segment destination for Data Export.
> 
> For information about the Experimentation integration for Segment, read [Sending Segment events to experiments](/home/experimentation/segment).
> 

Reviewer: ```suggestion
This topic explains how to create and test a Segment destination for Data Export.

<Callout intent="info">
<CalloutTitle>Experimentation integration for Segment</CalloutTitle>
<CalloutDescription>

For information about the Experimentation integration for Segment, read [Sending Segment events to experiments](/home/experimentation/segment).

</CalloutDescription>
</Callout>

```
